frontend/include/chpl/resolution/resolution-types.h

```diff
@@ -885,6 +885,103 @@ class TypedFnSignature {
   /// \endcond DO_NOT_DOCUMENT
 };
 
+// Container for resolution candidates and (if applicable) their correponding
+// forwarding-to types.
+struct CandidatesAndForwardingInfo {
+  private:
```

Comment:
```suggestion
 private:
```

We 1-dedent `public` etc. in Dyno code, I believe.